### PR TITLE
Footsteps: Fixed silenced footsteps when moving fast while crouching

### PR DESCRIPTION
**Pull request by ZenBre4ker**
#810
Instead of limiting to the players max speed, which doesnt change while crouching, we now check for the real speed, which is 60 for crouching, 100 for slowwalking and 220 for normal walking. Everything below 150 is now muted.
If someone can bhop and is faster than 150 you can hear him. Therefore cant be abused anymore.

### CODE REVIEW

**Review by TimGoll — APPROVED** [review]
interesting, I think this is a good thing ~~, but you should add a changelog entry~~

**Review by saibotk — APPROVED** [review]
Thank you as always Sven. 

It is a pleasure working with you :rocket:

File: gamemodes/terrortown/gamemode/shared/sh_main.lua [hunk 1]
```diff
@@ -8,6 +8,8 @@ local UpdateSprint = UpdateSprint
 
 local MAX_DROWN_TIME = 8
 
+local sneakSpeedSquared = math.pow(150,2)
```
Comment (Alf21) [inline gamemodes/terrortown/gamemode/shared/sh_main.lua:11]:
```suggestion
local sneakSpeedSquared = math.pow(150, 2)
```

**Review by Alf21 — APPROVED** [review]

**Comment by ZenBre4ker** [comment]
done ^^

**Pull request merged**